suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -40,15 +40,23 @@ describe('Model Integration Test', () => {
   })
 
   test('Create a ModelInstanceDocument on one node and read it from another', async () => {
+    await waitForIndexingOrTimeout(
+      ceramicNode1,
+      ceramicNode2,
+      modelId,
+      1000 * 60 * indexWaitTimeMin,
```

Comment:
does this timeout need to be so long?  Can we just re-use the nodeSyncWaitTimeSec?  It makes sense to use the same timeout here since this is also just waiting on a single event to sync between two nodes.

Reply:
yeah, the events are different however imo. Start indexing call would be way less frequent compared to a call to load the document. It would make sense if we guaranteed an SLA of about 1 minute for the node to actually get data from an interest after it has declared it. 
Don't feel too strongly about it either way. Is there an advantage in wanting to simply this to share same timeouts other than code reusability which I am overlooking?